apps/issuer/serializers_v1.py

```diff
@@ -223,6 +225,122 @@ class BadgeClassExpirationSerializerV1(serializers.Serializer):
     duration = serializers.ChoiceField(source='expires_duration', allow_null=True,
                                        choices=BadgeClass.EXPIRES_DURATION_CHOICES)
 
+# https://stackoverflow.com/questions/50973569/django-rest-framework-relatedfield-cant-return-a-dict-object
+
+class ModifiedRelatedField(serializers.RelatedField):
+    def get_choices(self, cutoff=None):
```

Comment:
To me it's not intuitive what exactly this method is doing (ordering something?) and what it's returning (sometimes an empty object, sometimes an ordered dictionary?). Maybe both a comment documenting this and a return type definition would be nice. And maybe it makes sense to not return an empty object, but rather an empty `OrderedDict`?